igvc_navigation/src/swerve_drive/swerve_drive.h

```diff
@@ -0,0 +1,36 @@
+/**
+ * Converts Twist messages to velocity quads for the motors of a swerve drive robot
+ */
+
+#ifndef SWERVEDRIVE_H
+#define SWERVEDRIVE_H
+
+#include <geometry_msgs/Twist.h>
+#include <igvc_msgs/velocity_quad.h>
+#include <ros/ros.h>
+
+class SwerveDrive
+{
+public:
+  SwerveDrive();
+
+private:
+  ros::NodeHandle nh;
+  ros::NodeHandle pNh;
+
+  ros::Subscriber mbf_twist_;
+  ros::Publisher vel_pub_;
+
+  std::array<std::array<double, 2>, 4> positions_list;
+  std::array<std::array<double, 2>, 4> limits_list;
+  std::array<std::array<double, 2>, 4> wheel_info;
+  double max_vel_{};
```

Comment:
Is there a particular reason for zero initializing `max_vel_`?

Reply:
Not really. The assertions gives it a default value anyways. I'll just remove it.